chapters/synchronous.tex

```diff
@@ -778,7 +778,8 @@ \subsection{Sub-clock conversion operators}\label{sub-clock-conversion-operators
 \\ \hline
 \lstinline!noClock(u)!
 &
-The clock of \lstinline!y = noClock(u)! is always inferred. At every tick
+The clock of \lstinline!y = noClock(u)! is always inferred; \lstinline!u! and \lstinline!y!
+must be part of the same base-clock. At every tick
```

Comment:
For some reason, I think of the semicolon here as beginning some kind of consequence or explanation, which I don't think is the intention.  How about _and_?
```suggestion
The clock of \lstinline!y = noClock(u)! is always inferred, and \lstinline!u! must be part of the same base-clock as \lstinline!y!. At every tick
```